Lil-Peep/src/scripts/site.js

```diff
@@ -0,0 +1,7 @@
+function openNav() {
+   document.getElementById("mySidenav").style.width = "90%";
```

Comment:
This was already commented before... use single quotes instead of double quotes... and try to use CSS classes instead of modifying/setting inline styles

Reply:
@hrodriguez1 can you change double quotes to single quotes, please?